ui/src/composables/private/use-field.js

```diff
@@ -529,14 +529,17 @@ export default function (state) {
   }
 
   // expose public methods
-  Object.assign(proxy, { focus, blur })
+  Object.assign(proxy, {
+    focus: proxy.focus || focus,
+    blur
+  })
 
   onMounted(() => {
     if (isRuntimeSsrPreHydration === true && props.for === void 0) {
       state.targetUid.value = getTargetUid()
     }
 
-    props.autofocus === true && focus()
+    props.autofocus === true && proxy.focus()
```

Comment:
In the previous change you're looking if proxy.focus() exists, else use the local focus(). Here you assume proxy.focus() always exists. Which behavior is right?

Reply:
Exactly. If it didn't exist before it is the local one for sure.

The problem is that I'm not sure it's the food solution and not only a patch.
It would mean we have to add checks to all the use files for all the things added in proxy and  in state.
Maybe it would be better to move the invocation of the use files above the assign to state and proxy?